### PR TITLE
Seller registration

### PR DESCRIPTION
**Pull request by akanshajuyal**
Seller Registration

### CODE REVIEW

File: src/components/RegisterForm.component/SellerRegistration.js [hunk 1]
```diff
@@ -1,124 +1,142 @@
...
 export const SellerRegistration = () => {
-  const {
-    register,
-    handleSubmit,
-    formState: { errors, isDirty, isValid },
+    const [values, setValues] = useState({
+        email: "",
+        password: "",
+        confirmPassword: "",
+        phoneNumber: "",
```
Comment (palvi8) [inline src/components/RegisterForm.component/SellerRegistration.js]: let's set the initial values null instead ""

File: src/components/RegisterForm.component/SellerRegistration.js [hunk 2]
```diff
@@ -1,124 +1,142 @@
...
+
+    const [errors, setErrors] = useState({})
+
+    const handleChange = (e) => {
+        setErrors(validation(values, e.target['name']))
+        setValues({
+            ...values,
+            [e.target.name]: e.target.value,
+        })
+        console.log(errors)
```
Comment (palvi8) [inline src/components/RegisterForm.component/SellerRegistration.js]: we can remove this console

File: src/components/RegisterForm.component/SellerRegistration.js [hunk 3]
```diff
@@ -1,124 +1,142 @@
...
-          <div style={{ height: '36px' }}>
-            {errors.email && <p>{errors.email.message}</p>}
-          </div>
-        </div>
+            <div className="form-inputs">
+                <label>Email address</label>
+                <input
+                    type="email"
+                    name="email"
+                    value={values.email}
```
Comment (palvi8) [inline src/components/RegisterForm.component/SellerRegistration.js]: please add autoComplete="off" wrt each input field

File: src/components/RegisterForm.component/SellerRegistration.js (hunk 1 above)
Comment (akanshajuyal) [reply src/components/RegisterForm.component/SellerRegistration.js]: Done

File: src/components/RegisterForm.component/SellerRegistration.js [hunk 4]
```diff
@@ -1,124 +1,145 @@
...
+import {useDispatch} from "react-redux";
+
+import {Link} from "react-router-dom";
 
 export const SellerRegistration = () => {
-  const {
-    register,
-    handleSubmit,
-    formState: { errors, isDirty, isValid },
+    const [values, setValues] = useState({
```
Comment (palvi8) [inline src/components/RegisterForm.component/SellerRegistration.js:9]: lets rename this variable from `value` to something more meaningful, same for setValues method

**Pull request closed without merging**